src/main/java/com/netflix/imflibrary/st0377_41/MCAContent.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *
+ * Copyright 2024 RheinMain University of Applied Sciences, Wiesbaden, Germany.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+
+package com.netflix.imflibrary.st0377_41;
+
+/**
+ * This enum lists the MCA Content Vocabulary defined in the following specification:
+ * ST 377-41:2021 Table 2
```

Comment:
Unless I'm missing a reason not to do that, let's use the updated list from ST 377-41:**2023**. 

While the normative reference from ST 2127-1 points at the 2021 version of ST 377-41, it appears to be only referenced in this statement: "Application standards **should** adopt the MCA Controlled Vocabulary as defined in SMPTE ST 377-41.".

Reply:
Modified accordingly, for your review!